migrations/versions/test_migration_0.py

```diff
@@ -0,0 +1,4 @@
+# Migration script generated for test_migration_0
+# Current version: 0
```

Comment:
best to test what we're using.  generate a random `uuid.uuid4()` and save in your test_migration file for testing, rather than simple ints, to avoid surprises.